react/src/pages/utils/functions.tsx

```diff
@@ -251,3 +252,20 @@ export function createFieldObj(
         disableEdit: disableEdit,
     };
 }
+
+export function createEmptyRows(amount?: number): DataEntryRow[] {
+    if (!amount || amount < 1) amount = 1;
+
+    var arr = [];
+    for (let i = 0; i < amount; i++) {
+        arr.push({
+            family_codename: "",
+            participant_codename: "",
+            participant_type: "",
+            tissue_sample_type: "",
+            dataset_type: "",
+            condition: "GermLine",
```

Comment:
I'll wait until standup but I'm not sure if this is the best default choice, if there even should be one.